httpclient-processor/src/main/java/org/springframework/cloud/stream/module/http/HttpClientProcessorProperties.java

```diff
@@ -0,0 +1,63 @@
+package org.springframework.cloud.stream.module.http;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.expression.Expression;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+
+/**
+ * Configuration properties for the Http Client Processor module.
+ *
+ * @author Waldemar Hummer
+ */
+@ConfigurationProperties
+public class HttpClientProcessorProperties {
```

Comment:
I think for the expression-based properties we probably still do want to support simple literals (i.e. not requiring the user to wrap literals with quotes). For example, we should have mutually exclusive properties `body` and `bodyExpression`. Sorry that was not clear when I originally suggested relying on Expressions. I can make that change while merging, though.